File: Oke_server/oke/src/main/java/com/bnuz/oke/entity/Prize.java
package com.bnuz.oke.entity;

/**
 * Prize
 *
 * @author handsome
 * @date 2020年 05月15日 00:16:04
 */
public class Prize {

	private int prizeId;

	private Scratcher scratcher;

	private String prizeDescribe;

	private int prizeCount;

	public int getPrizeId() {
		return prizeId;
	}

	public void setPrizeId(int prizeId) {
		this.prizeId = prizeId;
	}

	public Scratcher getScratcher() {
		return scratcher;
	}

	public void setScratcher(Scratcher scratcher) {
		this.scratcher = scratcher;
	}

	public String getPrizeDescribe() {
		return prizeDescribe;
	}

	public void setPrizeDescribe(String prizeDescribe) {
		this.prizeDescribe = prizeDescribe;
	}

	public int getPrizeCount() {
		return prizeCount;
	}

	public void setPrizeCount(int prizeCount) {
		this.prizeCount = prizeCount;
	}

	@Override
	public String toString() {
		return "Prize{" +
				"prizeId=" + prizeId +
				", scratcher=" + scratcher +
				", prizeDescribe='" + prizeDescribe + '\'' +
				", prizeCount=" + prizeCount +
				'}';
	}
}

File: Oke_client/Oke_teacher/Oke_teacher/Entity/Prize.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Oke_teacher.Entity
{
    /// <summary>
    /// 奖品信息封装类
    /// </summary>
    [Serializable]
    public class Prize
    {
        //奖品ID
        public int prizeId { set; get; }
        //所属抽奖
        public Scratcher scratcher { set; get; }
        //奖品描述
        public string prizeDescribe { set; get; }
        //奖品数量
        public int prizeCount { set; get; }
    }
}
